elyra/metadata/schemas/kfp.json

```diff
@@ -21,33 +21,51 @@
       "properties": {
         "description": {
           "description": "The description of this KFP runtime instance",
-          "type": "string"
+          "type": "string",
+          "uihints": {
+            "label": "Description"
+          }
         },
         "api_endpoint": {
           "description": "The API endpoint",
           "type": "string",
-          "format": "uri"
+          "format": "uri",
+          "uihints": {
+            "label": "API endpoint"
+          }
         },
         "cos_endpoint": {
           "description": "The Cloud Object Storage endpoint",
           "type": "string",
-          "format": "uri"
+          "format": "uri",
+          "uihints": {
+            "label": "Cloud Object Storage endpoint"
+          }
```

Comment:
It appears as though the property name portion of the label is lowercased and, IMO, should be capitalized.  Note that should #781 be used to address the label, these are capitalized.

Reply:
Fair enough, iirc I just copied the deception and removed the "the" for each of these. I think I meant to fox them later and forgot. Either way this should wait on #781 anyway